Polygon_mesh_processing/test/Polygon_mesh_processing/triangulate_faces_hole_filling_all_search_test.cpp

```diff
@@ -0,0 +1,4 @@
+// test with hole filling without dt3
+#define CGAL_TRIANGULATE_FACES_DO_NOT_USE_CDT2 1
+#define CGAL_HOLE_FILLING_DO_NOT_USE_DT3 1
+#include "triangulate_faces_test.cpp"
```

Comment:
Actually, the .cpp file is required to have the word `main` in its text, otherwise it will be ignored by our testsuite! :-/

I suggest you add a comment, like:
```C++
// The main() function is included by the line above.
```
That will be sufficient for the test script.

Reference:
https://github.com/CGAL/cgal/blob/4adbbdcab65f4d39a7c3dac72485c39c3d4fb227/Scripts/developer_scripts/create_cgal_test#L160-L178

Reply:
cool, thanks!